lib/api_client.ex

```diff
@@ -3,22 +3,66 @@ defmodule GlassFactoryApi.ApiClient do
   Client which make external HTTP Request
   """
 
-  alias GlassFactoryApi.HttpAdapter
+  alias GlassFactoryApi.Member
 
   @access_info Application.get_env(:glass_factory_api, GlassFactoryApi.ApiClient)
 
-  @type http_response() :: {atom(), HttpAdapter.response()}
+  @doc """
+  List the members of a organization
+
+  ## Examples
+
+      iex> GlassFactoryApi.Member.list()
+      [ %Member{
+          name: "John Doe",
+          email: "john.doe@example.org",
+          id: 0,
+          archived: false,
+          capacity: 8.0,
+          freelancer: false,
+          joined_at: "2019-01-01"
+        }
+      ]
+  """
```

Comment:
As this function is really common and easy to understand, IMO you can remove the comments...

Or, if you prefer to keep it, the function name is wrong, right? It must be `list_members()` instead of just `list()`